package.json

```diff
@@ -11,12 +11,12 @@
   "private": true,
   "dependencies": {
     "@angular/animations": "^16.2.0",
-    "@angular/cdk": "^16.2.7",
+    "@angular/cdk": "^16.2.12",
```

Comment:
I'd rather not make these version updates to angular/cdk and/or angular/material below.